src/stores/FilterBarStore.js

```diff
@@ -78,6 +79,10 @@ export class FilterBarStore {
     return this.exportResultsUrl;
   }
 
+  hasExportAllOptions() {
+    return this.exportAllOptions === 'true';
```

Comment:
do you use this method?

Reply:
No, good pick up, fixed: https://github.com/rdytech/react-filterbar/pull/138/commits/be92fb9cb570573e819ed8f17fc9c06cfea5f458